prbt_ikfast_manipulator_plugin/package.xml

```diff
@@ -22,16 +22,20 @@
   </export>
 
   <build_depend>moveit_core</build_depend>
+  <build_depend>moveit_ros_planning</build_depend>
   <build_depend>liblapack-dev</build_depend>
   <build_depend>roscpp</build_depend>
   <build_depend>tf2_kdl</build_depend>
   <build_depend>pluginlib</build_depend>
+  <build_depend>tf2_eigen</build_depend>
+  <build_depend>eigen_conversions</build_depend>
   <run_depend>moveit_core</run_depend>
+  <run_depend>moveit_ros_planning</run_depend>
```

Comment:
I saw a commit removing this dependency. Which moveit version did you have when you ran the update? current is 1.0.1

Reply:
This is necessary for the RobotModelLoader

Reply:
Maybe this is a good point for migrating from package format 1.0 -> 2.0? Then it would suffice to declare `moveit_ros_planning` as test_depend

Reply:
no according to catkin_lint. You still need it as build_depend. I switched to 2.0 now, because this is what the moveit template / script expects